discovery-provider/requirements.txt

```diff
@@ -13,15 +13,27 @@ pylint==2.3.1
 ipfshttpclient==0.6.0
 pytest-cov==2.6.0
 pytest-dotenv==0.5.2
-base58==1.0.0
 flask-cors==3.0.6
 gunicorn==20.0.4
 jsonschema==3.2.0
 flask-restx==0.2.0
 hashids==1.2.0
 fakeredis==1.4.2
 jsonformatter==0.3.0
-py-cid==0.3.0
 pytest-postgresql==2.4.1
 eventlet==0.28.0
 psutil==5.8.0
+
+base58==2.1.0
+certifi==2020.12.5
+cffi==1.14.5
+chardet==4.0.0
+construct==2.10.60
+idna==2.10
+pycparser==2.20
+PyNaCl==1.4.0
+requests==2.25.1
+six==1.15.0
+solana==0.6.5
+typing-extensions==3.7.4.3
+urllib3==1.26.3
```

Comment:
We should probably add a comment mentioning why we are separating this out (Solana dependencies), or merge this into one section.

Reply:
agreed, good point